data/migrations/V0047__alter_ofec_sched_a_aggregate_state_recipient_totals_mv.sql

```diff
@@ -0,0 +1,177 @@
+            pacs_totals.state_full,
+            pacs_totals.committee_type,
+            pacs_totals.committee_type_full
+           FROM pacs_totals
+        UNION ALL
+         SELECT overall_total.total,
+            overall_total.count,
+            overall_total.cycle,
+            overall_total.state,
+            overall_total.state_full,
+            overall_total.committee_type,
+            overall_total.committee_type_full
+           FROM overall_total
+  ORDER BY 4, 3, 6
+        )
+ SELECT row_number() OVER () AS idx,
+    combined.total,
+    combined.count,
+    combined.cycle,
+    combined.state,
+    combined.state_full,
+    combined.committee_type,
+    combined.committee_type_full
+   FROM combined
+WITH DATA;
+
+ALTER TABLE public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  OWNER TO fec;
+GRANT ALL ON TABLE public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP TO fec;
+GRANT SELECT ON TABLE public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP TO fec_read;
+
+
+CREATE INDEX ofec_sched_a_aggregate_state_recip_committee_type_full_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (committee_type_full COLLATE pg_catalog."default", idx);
+
+
+CREATE INDEX ofec_sched_a_aggregate_state_recipient__committee_type_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (committee_type COLLATE pg_catalog."default", idx);
+
+
+CREATE INDEX ofec_sched_a_aggregate_state_recipient_tota_state_full_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (state_full COLLATE pg_catalog."default", idx);
+
+
+CREATE INDEX ofec_sched_a_aggregate_state_recipient_totals_mv_count_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (count, idx);
+
+
+CREATE INDEX ofec_sched_a_aggregate_state_recipient_totals_mv_cycle_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (cycle, idx);
+
+
+CREATE INDEX ofec_sched_a_aggregate_state_recipient_totals_mv_state_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (state COLLATE pg_catalog."default", idx);
+
+
+CREATE UNIQUE INDEX ofec_sched_a_aggregate_state_recipient_totals_mv_tmp_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (idx);
+
+
+CREATE INDEX ofec_sched_a_aggregate_state_recipient_totals_mv_total_idx_TMP
+  ON public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP
+  USING btree
+  (total, idx);
+
+-- ------------------------------------------------------
+DROP MATERIALIZED VIEW IF EXISTS public.ofec_sched_a_aggregate_state_recipient_totals_mv;
+
+ALTER MATERIALIZED VIEW IF EXISTS public.ofec_sched_a_aggregate_state_recipient_totals_mv_TMP RENAME TO ofec_sched_a_aggregate_state_recipient_totals_mv;
+
+--SELECT public.rename_indexes ('ofec_sched_a_aggregate_state_recipient_totals_mv');

```

Comment:
Did this not work for you?  Are there any questions in using it?  I would think we'd still be able to leverage it here to avoid having to write all of the index names by hand below, but wanted to double check and see.